src/Rules.Framework/RulesEngine.cs

```diff
@@ -64,6 +64,39 @@ public Task<RuleOperationResult> AddRuleAsync(Rule<TContentType, TConditionType>
             return this.AddRuleInternalAsync(rule, ruleAddPriorityOption);
         }
 
+        /// <summary>
+        /// PGet the unique condition types associated with rules of a specific content type/>.
```

Comment:
We may have a misspelling PGet on the documentation, right?

Reply:
Thank you. This type has been fixed